components/frontend/src/components/session/ask-user-question.tsx

```
@@ -0,0 +1,363 @@
+              <Input
+                autoFocus
+                placeholder="Type your answer..."
+                value={otherText[q.question] || ""}
+                onChange={(e) => handleOtherTextChange(q.question, e.target.value)}
+                onClick={(e) => e.stopPropagation()}
+                disabled={disabled}
+                className="mt-1 h-7 text-sm"
+              />
+            )}
+          </div>
+        </label>
```

Biome (2.4.6)

[error] A form label must be associated with an input.

(lint/a11y/noLabelWithoutControl)